card/mod/notifications/set/all/show_notifications.rb

```diff
@@ -0,0 +1,7 @@
+format :html do
+  view :flash do
+    flash_notice = Env.success.params[:flash]
```

Comment:
given the easy write API, I might expect Env.success.flash (with no args) to return this value.